### PR TITLE
owner(cdc): add owner for cdc v2

### LINKED ISSUE #10033: owner(cdc): add owner for cdc v2
owner(cdc): add owner for cdc v2

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: close #10033

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

File: cdcv2/owner/owner.go [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+	"github.com/pingcap/tiflow/pkg/config"
+	cdcContext "github.com/pingcap/tiflow/pkg/context"
+	cerror "github.com/pingcap/tiflow/pkg/errors"
+	"github.com/pingcap/tiflow/pkg/upstream"
+	"go.uber.org/zap"
+	"gorm.io/gorm"
+)
+
+// Impl implements the owner interface.
+type Impl struct {
```
Comment (3AceShowHand) [inline cdcv2/owner/owner.go:39]: How about rename this to `Owner`? 

I think the interface name should be `Owner`, and the implementation name is `owner`

**Review by 3AceShowHand — APPROVED** [review]

File: cdcv2/owner/owner.go (hunk 1 above)
Comment (sdojjy) [reply cdcv2/owner/owner.go:39]: owner is a part of the package name,  can not pass the lint checking

**Review by 3AceShowHand — APPROVED** [review]

File: cdcv2/owner/owner.go (hunk 1 above)
Comment (sdojjy) [reply cdcv2/owner/owner.go]: done

**Pull request merged**